pride/src/main/java/com/prezi/pride/cli/model/ProjectModelAccessor.java

```diff
@@ -59,7 +59,8 @@ public PrideProjectModel execute(File moduleDirectory, ProjectConnection connect
 
 					return builder.get();
 				} catch (Exception ex) {
-					throw new PrideException("Could not parse module in " + moduleDirectory + ": " + ex, ex);
+					throw new PrideException("Could not evaluate module in " + moduleDirectory +
+							". Run './gradlew' in the pride root to get more information about the error: " + ex, ex);
```

Comment:
If you want to show a more understandable error message, maybe we could collect the exceptions that happen properly, and show them here. They are already shown as part of the log, no?